src/renderer/components/dialog/publicationInfos/PublicationInfo.tsx

```diff
@@ -205,15 +204,25 @@ class PublicationInfo extends React.Component<IProps, IState> {
         return (
             <Dialog open={true} close={this.props.closeDialog}>
                 {this.state.coverZoom ?
-                    <Cover publication={publication} coverTypeUrl="coverUrl" onclick={() => this.setState({
-                        coverZoom: !this.state.coverZoom,
-                    })} />
+                    <Cover
+                        publication={publication}
+                        coverTypeUrl="coverUrl"
+                        onclick={this.coverOnClick}
+                        onKeyPress={this.coverOnKeyPress}/>
```

Comment:
Nit-picking :)
Let's choose either:
* `onKeyPress` and `onClick`
or:
* `onkeypress` and `onclick`
(but let's try not to mix case conventions)

Reply:
fix